nucypher-core-python/nucypher_core/__init__.py

```diff
@@ -29,4 +29,23 @@
     SessionStaticSecret,
     SessionSecretFactory,
     encrypt_for_dkg,
+    UserOperation,
+    UserOperationSignatureRequest,
+    PackedUserOperation,
+    PackedUserOperationSignatureRequest,
+    SignatureResponse,
+    deserialize_signature_request,
 )
+
+# Constants for signature request types
+class SignatureRequestType:
+    """Constants for signature request types."""
+    USEROP = 0
+    PACKED_USER_OP = 1
```

Comment:
Nitpicking: `USEROP` or `USER_OP`? Just to keep it consistent with `PACKED_USER_OP`.